### PR TITLE
package/msys: Add PKGBUILDs.

### PR DESCRIPTION
**Pull request by williamjcm**
And associated `.gitattributes` file.

Once again, comments and feedback are welcome! While mosra/corrade#56 and mosra/magnum#307 allowed me to do less mistakes this time, there might still be a few issues remaining.

### CODE REVIEW

File: package/msys/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+magnum/PKGBUILD text eof=lf
```
Comment (mosra) [inline package/msys/.gitattributes]: `magnum-plugins/PKGBUILD` :grin:

File: package/msys/magnum-plugins/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+        ${MINGW_PREFIX}/bin/cmake \
+            -G'Ninja' \
+            -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+            -DWITH_ASSIMPIMPORTER=OFF \
+            -DWITH_DDSIMPORTER=ON \
+            -DWITH_DEVILIMAGEIMPORTER=ON \
+            -DWITH_DRFLACAUDIOIMPORTER=ON \
+            -DWITH_DRWAVAUDIOIMPORTER=ON \
+            -DWITH_FREETYPEFONT=ON \
```
Comment (mosra) [inline package/msys/magnum-plugins/PKGBUILD]: Oh, interesting, HarfBuzz has a MSYS package too? Then I would do `WITH_HARFBUZZFONT=ON` here as well (and the corresponding dependency above).

The reason I did this in the Arch PKGBUILD was originally to reduce the amount of dependencies, but harfbuzz is pretty ubiquitous nowadays, so I think I'll enable it on Arch as well.

File: package/msys/.gitattributes (hunk 1 above)
Comment (williamjcm) [reply package/msys/.gitattributes]: 🤦‍♂️ 

I knew I forgot something.

File: package/msys/magnum-plugins/PKGBUILD (hunk 2 above)
Comment (williamjcm) [reply package/msys/magnum-plugins/PKGBUILD]: > HarfBuzz has a MSYS package too?

Yeah, it does. It's possible I mentioned it on Gitter months ago, back when I started with Magnum, but I could be wrong.

**Comment by williamjcm** [comment]
Should be good now. 😅

**Comment by mosra** [comment]
Merged in 294ff831316289339a4f6f3dcc14eb402ae0c3ed, thank you!

**Pull request closed without merging**